<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>855</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>855</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>832</width>
        <height>537</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="videoTools">
       <attribute name="title">
        <string>VideoTools</string>
       </attribute>
       <widget class="QPushButton" name="pButSequenceImageDir">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>220</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSequenceImageDir">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_43">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Sequence Dir:</string>
        </property>
       </widget>
       <widget class="Line" name="line_18">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_55">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Common Configs</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditPrefix">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_50">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Prefix:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxSuffix">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>.jpg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.png</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.jpeg</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_51">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Suffix:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxZfill">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>50</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Zfill:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pButSeq2Video">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>290</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Make Video</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pButAnnotationPath">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>250</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAnnotationPath">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>250</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>40</y>
          <width>20</width>
          <height>461</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_19">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_56">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>340</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Video to Sequence</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Video Path:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pButVideoPath">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>390</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditVideoPath">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>390</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pButVideo2Sequence">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>430</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Extract Frames</string>
        </property>
       </widget>
       <widget class="Line" name="line_20">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_57">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>170</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Sequence to Video</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxStartIdx">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>50000</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxInterval">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>140</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Interval:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxEndIdx">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>110</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>50000</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>110</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>End:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxFPS">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>80</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>70</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>80</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>FPS:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxAnnotation">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Detection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tracking</string>
         </property>
        </item>
       </widget>
       <widget class="Line" name="line_21">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>20</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pButViewTracks">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>View Tracks</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxLineWidth">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>140</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>140</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>LineWidth:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="sequenceTools">
       <attribute name="title">
        <string>Other</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
